apps/web/lib/video/[uid]/getServerSideProps.ts

```diff
@@ -26,6 +27,14 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
     bookingUid: context.query.uid as string,
   });
 
+  // Below if block is for local testing purposes only
+  // STARTS------------------------------------------------------------------------------
+  if (booking?.references[0] && isENVDev) {
+    // meetingUrl is `null` in dev env, so setting a dummy meetingUrl (it's a past but real meeting link in production env)
+    booking.references[0].meetingUrl = "https://app.cal.com/video/wvsSuNC8r9ZScPhA6JKpiL";
```

Comment:
@hbjORbj why are you using this url? this would get expired after few weeks

Reply:
what url do you suggest we use here for testing? if there is a permanently valid link you know we could use here, that would be amazing

Reply:
I have updated expiry time of this room

Reply:
oh wow, how?

Reply:
Should this be an ENV variable? People can exploit this URL

Reply:
Opening a PR for this